test/projectTest.spec.js

```diff
@@ -112,7 +112,7 @@ describe('developer', () => {
         proj1.addDeveloper(dev1);
         proj1.addDeveloper(dev2);
         proj1.addDeveloper(dev3);
-        assert.isTrue(proj1.weekColorCode(20) === 'GREEN');
+        assert.isTrue(proj1.weekColorCode(20) === 'week-green');
```

Comment:
For these asserts i would probably use `.equal(a, b)`

Reply:
Har ikke skrevet testen

Reply:
Ah i see